tests/node/app-boot-test.js

```diff
@@ -3,19 +3,148 @@
 var path = require('path');
 var distPath = path.join(__dirname, '../../dist');
 
+QUnit.config.notrycatch = true;
```

Comment:
Great catch; I did not mean to keep that.

Let me take this opportunity to rant about the fact that you can have a completely wrong set of npm packages installed that do not satisfy the dependencies in your `package.json` and neither node nor npm will do anything to warn you, just choosing to require the wrong package. This is debugging detritus from us trying to explain why the tests were passing on one computer but inexplicably not passing on another, and it ended up being us forgetting to `npm install`. Isn't this what computers are supposed to be good at?